src/main/java/seedu/phu/ui/ViewCommandCard.java

```diff
@@ -56,21 +61,43 @@ public ViewCommandCard(Internship internship, int displayedIndex) {
         super(FXML);
         this.internship = internship;
         name.setText(internship.getName().fullName);
-        phone.setText(internship.getPhone().value);
-        remark.setText(internship.getRemark().value);
-        email.setText(internship.getEmail().value);
-        position.setText(internship.getPosition().positionName);
+        phone.setText("phone: " + internship.getPhone().value);
```

Comment:
Nitpicking but maybe can change the first letter to uppercase so looks neater

Reply:
good idea